index.css

```
@@ -1,179 +1,361 @@
+padding-bottom: 5px;
+}
+.the-end{
+    text-align: center;
+    background-color: #13883C;
+    width: 100%;
+    line-height: 40px;
+    color: white;
+}
+.grid-item22{
+    
+}
```

Biome (1.9.4)

[error] An empty block isn't allowed.

Consider removing the empty block or adding styles inside it.

(lint/suspicious/noEmptyBlock)